http_stubs/views.py

```diff
@@ -35,14 +36,21 @@ def _httpstub_executor(
     """
     sleep(stub.resp_delay / 1000)
 
+    request_body = _request_body_decode(request)
+
+    tpl_ctx = {'body': request_body}
+
+    for arg_name, arg_val in dict(request.GET).items():
+        tpl_ctx[arg_name] = arg_val[0]
```

Comment:
```suggestion
    tpl_ctx = dict(request.GET)
    tpl_ctx['body'] = request_body
```

Reply:
Thanks! But this way is incorrect. Each value from request.GET is one element list.